### PR TITLE
Use 'backup_image_after_testing_passed' to backup image

### PR DESCRIPTION
**Pull request by yanan-fu**
id: 1683557
Signed-off-by: Yanan Fu <yfu@redhat.com>

### CODE REVIEW

**Comment by yanan-fu** [comment]
depends on https://github.com/avocado-framework/avocado-vt/pull/1978

**Comment by yanan-fu** [comment]
@luckyh @hereischen  call for your review, thanks !

File: qemu/tests/cfg/rh_kernel_update.cfg [hunk 1]
```diff
@@ -1,12 +1,12 @@
-- rh_kernel_update:
+ rh_kernel_update:
```
Comment (hereischen) [inline qemu/tests/cfg/rh_kernel_update.cfg]: delete `-` by mistake?

Comment (sathnaga) [reply qemu/tests/cfg/rh_kernel_update.cfg]: @yanan-fu `backup_image` rename can be avoided, rather as suggested we can keep the same in avocado-vt patch.

Comment (hereischen) [inline qemu/tests/cfg/rh_kernel_update.cfg]: `-` deleted by mistake?

**Comment by hereischen** [comment]
@sathnaga sorry, did not mean to delete your comment, just found my comment was no correct.

**Comment by sathnaga** [comment]
we need not have this param name change, rather backup_image itself we can default it only for test_passed.

File: qemu/tests/cfg/rh_kernel_update.cfg (hunk 1 above)
Comment (yanan-fu) [reply qemu/tests/cfg/rh_kernel_update.cfg]: Yes,  update to correct it, thanks!

**Comment by yanan-fu** [comment]
> we need not have this param name change, rather backup_image itself we can default it only for test_passed.

Thanks sathnaga,  let's focus on the avocado-vt patch, then this patch will follow it.

**Comment by luckyh** [comment]
Thanks all.

**Pull request merged**